File: go/1117.go
package main

import (
	"fmt"
)

func main() {
	var n float64
	var soma float64 = 0.0

	for i := 1; i <= 2; {
		fmt.Scan(&n)
		if n >= 0 && n <= 10 {
			soma += n
			i++
		} else {
			fmt.Println("nota invalida")
		}
	}

	media := soma / 2
	fmt.Printf("media = %.2f\n", media)
}

File: Python/1 - Iniciante/1117.py
# coding: UTF-8
# Arquivo: 1117

# Funcional no beecrowd

i = 1
soma = 0
while i <= 2:
    n = float(input())
    if n >= 0 and n <= 10:
        soma += n
        i += 1
    else:
        print("nota invalida")

media = soma / 2
print(f"media = {media:.2f}")
# Fim